src/App.jsx

```diff
@@ -0,0 +1,46 @@
+
+import {Route ,Routes} from 'react-router-dom'
+import {Home} from "./pages/home/Home.jsx"
+import { Builder } from './pages/builder/Builder';
+import Navbar from './components/organisms/Navbar';
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+const App = () => {
+  const menuItems = [
+    { icon: 'home', label: 'Home', to: '/home' },
+    { icon: 'ℹ️', label: 'About', to: '/Builder' },
+    // ... other menu items
```

Comment:
Let's please remove the comment 🙏🏼

We usually add comments to code if necessary, We should focus more on making our code readable and self explanatory. I don't think this part needs a comment 🙈